eng/Versions.props

```diff
@@ -36,6 +36,12 @@
     <!-- Dependencies from https://github.com/dotnet/cli -->
     <MicrosoftDotNetMSBuildSdkResolverPackageVersion>3.0.100-preview5.19218.1</MicrosoftDotNetMSBuildSdkResolverPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <NETStandardLibraryPackageVersion>2.1.0-prerelease.19208.5</NETStandardLibraryPackageVersion>
```

Comment:
@dsplaisted do you have context for this change? Should it be in master?

Reply:
No, it shouldn't.  It was deleted in this PR: https://github.com/dotnet/core-sdk/pull/1309